mlflow/genai/evaluation/base.py

```diff
@@ -279,22 +281,33 @@ def _evaluate_oss(data, scorers, predict_fn, model_id):
 
     # TODO: Log input dataset to the run
     with (
-        _start_run_or_reuse_active_run(),
+        _start_run_or_reuse_active_run() as run_id,
         _set_active_model(model_id=model_id) if model_id else nullcontext(),
     ):
         return harness.run(
             predict_fn=predict_fn,
             dataset=data,
             scorers=scorers,
+            run_id=run_id,
         )
-        # TODO: Support logging aggregated metrics to the logged model
 
 
 def _evaluate_dbx(data, scorers, predict_fn, model_id):
     """In Databricks, we run GenAI evaluation using databricks-agents package and
     the mlflow.evaluate() function. This is a temporary migration state and we will
     eventually unify this into OSS flow.
     """
+
+    # NB: The "RAG_EVAL_MAX_WORKERS" env var is used in the DBX agent harness, but is
+    # deprecated in favor of the new "MLFLOW_GENAI_EVAL_MAX_WORKERS" env var. The old
+    # one is not publicly documented, but we keep it for backward compatibility.
+    if MLFLOW_GENAI_EVAL_MAX_WORKERS.is_set() and os.environ.get("RAG_EVAL_MAX_WORKERS") is None:
```

Comment:
```suggestion
    if MLFLOW_GENAI_EVAL_MAX_WORKERS.is_set() and "RAG_EVAL_MAX_WORKERS" in os.environ.get("RAG_EVAL_MAX_WORKERS"):
```

because we care about the presence, not the value.